configs/sites/tier2/ubuntu2404-nvhpc/packages_nvhpc.yaml

```diff
@@ -0,0 +1,18 @@
+packages:
+  all:
+    compiler:: [nvhpc@25.1, gcc@13.3.0]
+    providers:
+      mpi:: [openmpi@4.1.5]
+      blas:: [nvhpc@25.1]
+      zlib-api:: [zlib]
+      fftw:: [nvhpc@25.1]
```

Comment:
For my education...
- I see you fixed the MPI version, thanks!
- You added `fftw` from `nvhpc` here but then specify `ecmwf-atlas ~fftw` below; how come? Does Nvidia's fft library not satisfy atlas' requirements?
- What changed here to allow `cublas` and `cufft` to be found at runtime? Was this about the drivers, or something else in the spack-stack setup?

Reply:
Thanks for the explanations!